test/setup.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+# ------------------------------------------------------------------
+# [korthout] Setup test repo to be used to test the backport action
+#
+# Produces a dir "repo" with history like:
+# 
+# (cd repo; git log --graph --oneline)
```

Comment:
the one thing that is a bit confusing here is that feature b seems to be created before feature a. Not sure if this is deliberate or not. I don't think it's a big deal in the overall picture though.

Reply:
Yeah it seems strange, that's just how git displays the log. Actually the commit `feat: add a` is created before `feat: add b`. For that we could add the `--author-date-order` flag. The output is then:

```
*-.   2a70db9 (HEAD -> master, release-2) Merge branches 'feature-b' and 'feature-c'
|\ \
| * | 832153c (feature-b) chore: add to b
| | * 5d0d08e (feature-c) chore: add to c
| * | 1d22a7e chore: add to b
| | * a9ee636 feat: add c
| |/
|/|
| * 17a42a4 feat: add b
* |   4c6eea4 (release-1) Merge branch 'feature-a'
|\ \
| |/
|/|
| * 898792b (feature-a) feat: add a
|/
* 30e1354 init: add README.md
```